### ISSUE TITLE
Explicit order of operations?

### ISSUE THREAD

**Issue by mleipold** [ParkerICI/premessa#43]
Hi,

Is there an explicit order of operations for the steps of Premessa? 

From the order in the Wiki, my assumption is that concatenation should happen before normalization.  And that's consistent with old how-to documents like the attached PDF (see page 7).

However, since that's the opposite of how the Fluidigm software workflow operates, it would be nice to have an explicit statement in the Premessa wiki on this......

Mike

[Normalization_EQ-4-Element-Beads_ug.pdf](https://github.com/ParkerICI/premessa/files/9011755/Normalization_EQ-4-Element-Beads_ug.pdf)

**Comment by SansMorel** [comment]
Hi, Mike

Regarding concatenation before normalization or vice versa I don't think this makes any difference. As I understand it the baseline used for normalization is calculated using all files either way. Clicking Normalize with the standard setting "Current files" in "Select baseline for normalization" leads to calling `calculate_baseline` with the files in the working directory. `calculate_baseline` then reads all fcs files, gates out the beads and grabs the total medium intensity of each bead channel.
